recipe/meta.yaml

```diff
@@ -1,36 +1,38 @@
 {% set name = "aiosqlite" %}
 {% set version = "0.18.0" %}
+{% set sha256 = "faa843ef5fb08bafe9a9b3859012d3d9d6f77ce3637899de20606b7fc39aa213" %}
```

Comment:
If the variable is used only once then it's better not to repeat it. Because instead of 1 line of code, you will have 2 lines. There are no advantages.